README.md

```diff
@@ -11,16 +11,82 @@ This repo should be used as a template when creating new repos for the Apprentic
 
 ## Developer Setup
 
+The portal uses azure storage to retrieve settings. To use local settings UseDevelopmentStorage=true should be set in the local appsettings.deverlopment.json file as detailed below. Repo das-employer-config section [das-apprentice-portal-web](https://github.com/SkillsFundingAgency/das-employer-config/tree/master/das-apprentice-portal-web) holds the settinge for the portal web.
+
+After cloning the portal repo it should build without any issues.  To run the portal web home page requires a running login service, commitments website & accounts website.  The config section 'ApplicationUrls' indicates where the service should point to (eg with the login service, locally https://localhost:5001 or https://login.at-aas.apprenticeships.education.gov.uk/ for the at environment ).  If you want to run all locally, then the login service, accounts website and commitments website all need to be running locally too. 
+
 ### Requirements
 
 ### Setup
 
-### Config
\ No newline at end of file
+### Config
+
+The portal webiste hosts the homepage and gets its configuration settings from an appsettings.json file.
+Create a local appsettings.development.json file as follows:
+
+    {
+      "Logging": {
+        "LogLevel": {
+          "Default": "Information",
+          "Microsoft": "Warning",
+          "Microsoft.Hosting.Lifetime": "Information"
+        }
+      },
+      "AllowedHosts": "*",
+      "ConfigurationStorageConnectionString": "UseDevelopmentStorage=true",
+      "ConfigNames": "SFA.DAS.ApprenticePortal.Web",
+      "EnvironmentName": "LOCAL",
+      "Cdn": {
+        "Url": "https://das-at-frnt-end.azureedge.net"
+      },
+      "GoogleAnalytics": {
+        "GoogleTagManagerId": "GTM-MP5RSWL"
+      },
+      "ApprenticePortalOuterApi": {
+        "ApiBaseUrl": "https://localhost:5123/",
+        "SubscriptionKey": "key",
+        "ApiVersion": "1"
+      },
+      "ApplicationUrls": {
+        //"ApprenticeHomeUrl": "https://portal.at-aas.apprenticeships.education.gov.uk/",
+        //"ApprenticeCommitmentsUrl": "https://commitments.at-aas.apprenticeships.education.gov.uk/",
+        //"ApprenticeLoginUrl": "https://login.at-aas.apprenticeships.education.gov.uk/",
+        //"ApprenticeLoginUrl": "https://accounts.at-aas.apprenticeships.education.gov.uk/",
+        "ApprenticeHomeUrl": "https://localhost:44398",
+        "ApprenticeCommitmentsUrl": "https://localhost:7070",
+        "ApprenticeLoginUrl": "https://localhost:5001",
+        "ApprenticeAccountsUrl": "https://localhost:7080"
+      },
+      "ZenDesk": {
+        "ZenDeskSnippetKey": "e0730bdd-a32c-4c39-8032-7d7a908eacb4",
+        "ZenDeskSectionId": "360003002699",
+        "ZenDeskCobrowsingSnippetKey": "Qec2OgXsUy8lsUO1iOuleFAMjl7AcIEGdOojlQjLesShBHx0wrM4xXlR7X1h9bEo"
```

Comment:
I think we should replace the values here with dummy ones

Reply:
done